src/experimental/build/commands/build/wasm_binary/rust/experimental_canister_template/src/internal_canister_methods.rs

```diff
@@ -62,11 +62,16 @@ fn _azle_reject_callbacks_len() -> i32 {
         let runtime = runtime.as_mut().unwrap();
```

Comment:
No need to do any checks in this file, we'll get an error in these cases anyway, and this is generally not developer-facing code